packages/mobile/src/helpers/price/check-slider-price-validity.ts

```diff
@@ -0,0 +1,18 @@
+import {
+  MINIMUM_SLIDER_PRICE,
+  MAXIMUM_SLIDER_PRICE,
+} from '~/common/constants/constants';
+
+const checkSliderPriceValidity = (price: string) => {
+  const priceAsNumber = +price.replace(/\D/gi, '');
+  if (priceAsNumber < MINIMUM_SLIDER_PRICE) {
```

Comment:
Maybe it's better to use explicit type casting?
`Number(price.replace(/\D/gi, ''))`

Reply:
changed